# already fixed upstream
Index: munin-1.2.6/node/node.d/apache_accesses.in
===================================================================
--- munin-1.2.6.orig/node/node.d/apache_accesses.in	2009-05-28 19:56:49.000000000 +0200
+++ munin-1.2.6/node/node.d/apache_accesses.in	2009-05-28 19:57:12.000000000 +0200
@@ -37,7 +37,7 @@
      env.ports 80
 
 The %d in the url will be replaced with the port.  The default port is
-80 as snown.
+80 as shown.
 
 The port list is a space separated list of ports.  NOTE that one
 single Apache can have several open ports, and the plugin needs only
Index: munin-1.2.6/node/node.d/apache_processes.in
===================================================================
--- munin-1.2.6.orig/node/node.d/apache_processes.in	2009-05-28 19:56:49.000000000 +0200
+++ munin-1.2.6/node/node.d/apache_processes.in	2009-05-28 19:57:16.000000000 +0200
@@ -36,7 +36,7 @@
      env.ports 80
 
 The %d in the url will be replaced with the port.  The default port is
-80 as snown.
+80 as shown.
 
 The port list is a space separated list of ports.  NOTE that one
 single Apache can have several open ports, and the plugin needs only
Index: munin-1.2.6/node/node.d/apache_volume.in
===================================================================
--- munin-1.2.6.orig/node/node.d/apache_volume.in	2009-05-28 19:56:49.000000000 +0200
+++ munin-1.2.6/node/node.d/apache_volume.in	2009-05-28 19:57:20.000000000 +0200
@@ -35,7 +35,7 @@
      env.ports 80
 
 The %d in the url will be replaced with the port.  The default port is
-80 as snown.
+80 as shown.
 
 The port list is a space separated list of ports.  NOTE that one
 single Apache can have several open ports, and the plugin needs only
