Type: bugfix

Not fixed upstream
Index: munin-1.2.6/node/node.d/exim_mailqueue.in
===================================================================
--- munin-1.2.6.orig/node/node.d/exim_mailqueue.in	2008-06-22 20:41:50.000000000 +0200
+++ munin-1.2.6/node/node.d/exim_mailqueue.in	2008-06-22 20:43:54.000000000 +0200
@@ -24,25 +24,26 @@
 #%# capabilities=autoconf
 
 DIRNAME=$(dirname $0)
-SPOOLDIR="unset"
-# You cannot trust the exit status of which
-EXIM=$(which exim 2>/dev/null)
-case $EXIM:$? in
-    *:1|no*) EXIM=$(which exim4 2>/dev/null)
-esac
-case $EXIM:$? in
-    *:1|no*) EXIM=''
-esac
+SPOOLDIR=${spooldir:-unset}
+EXIM=${exim:-unset}
+if [ "$EXIM" = "unset" ]; then
+    # You cannot trust the exit status of which
+    EXIM=$(which exim 2>/dev/null)
+    case $EXIM:$? in
+        *:1|no*) EXIM=$(which exim4 2>/dev/null)
+    esac
+    case $EXIM:$? in
+        *:1|no*) EXIM=''
+    esac
+fi
 
 GRAPHTITLE='Exim Mailqueue'
 
-SPOOLDIR=${spooldir:-unset}
-EXIM=${exim:-unset}
 QUEUEWARN=${queuewarn:-100}
 QUEUECRIT=${queuecrit:-200}
 GRAPHTITLE=${graphtitle:-$GRAPHTITLE}
 
-if [ "$SPOOLDIR" = "unset" ]
+if [ "$SPOOLDIR" = "unset" ] && [ -n "$EXIM" ]
 then
 	SPOOLDIR=$( ($EXIM -bP spool_directory | awk '{ print $3 "/input" }') 2>/dev/null)
 fi
