Index: munin-1.2.6/node/munin-run.in
===================================================================
--- munin-1.2.6.orig/node/munin-run.in	2009-02-26 22:55:18.000000000 +0100
+++ munin-1.2.6/node/munin-run.in	2009-02-26 22:55:59.000000000 +0100
@@ -97,9 +97,9 @@
              "paranoia!"    => \$paranoia,
              "help"         => \$do_usage );
 
-if ($do_usage)
+if ($do_usage or ! defined($ARGV[0]))
 {
-    print "Usage: $0 [options]
+    print "Usage: $0 [options] <plugin>
 
 Options:
     --help              View this message.
@@ -731,7 +731,7 @@
 =head1 DESCRIPTION
 
 Munin-node is a daemon that Munin connects to fetch data.  This data
-is stored in .rrd-files on the central server , and later graphed and
+is stored in .rrd-files on the central server, and later graphed and
 htmlified.
 
 munin-run is a perlscript to run the plugins used by the munin-node
