Index: munin-1.2.6/server/munin-graph.in
===================================================================
--- munin-1.2.6.orig/server/munin-graph.in	2008-10-13 08:09:43.000000000 +0200
+++ munin-1.2.6/server/munin-graph.in	2008-10-13 08:10:16.000000000 +0200
@@ -833,10 +833,10 @@
 	    my @complete = ();
 	    if ($RRDkludge) {
 		push (@complete,
- 		      '--font' ,'LEGEND:7:@@LIBDIR@@/VeraMono.ttf',
- 		      '--font' ,'UNIT:7:@@LIBDIR@@/VeraMono.ttf',
+ 		      '--font' ,'LEGEND:7:mono',
+ 		      '--font' ,'UNIT:7:mono',
 
- 		      '--font' ,'AXIS:7:@@LIBDIR@@/VeraMono.ttf');
+ 		      '--font' ,'AXIS:7:mono');
 	    }
 
 	    logger ("Processing $name -> $time") if $DEBUG;
