Index: munin-1.2.6/node/node.d.linux/df.in
===================================================================
--- munin-1.2.6.orig/node/node.d.linux/df.in	2008-06-27 16:47:10.000000000 +0200
+++ munin-1.2.6/node/node.d.linux/df.in	2008-06-27 16:47:51.000000000 +0200
@@ -79,7 +79,7 @@
 if ( $ARGV[0] eq "config" ) {
 
     # The headers
-    print "graph_title Disk usage (in %)\n";
+    print "graph_title Disk usage (in percent)\n";
     print "graph_args --upper-limit 100 -l 0\n";
     print "graph_vlabel %\n";
     print "graph_scale no\n";
Index: munin-1.2.6/node/node.d.linux/df_inode.in
===================================================================
--- munin-1.2.6.orig/node/node.d.linux/df_inode.in	2008-06-27 16:47:58.000000000 +0200
+++ munin-1.2.6/node/node.d.linux/df_inode.in	2008-06-27 16:48:19.000000000 +0200
@@ -80,7 +80,7 @@
 if ( $ARGV[0] eq "config" ) {
 
     # The headers
-    print "graph_title Inode usage (in %)\n";
+    print "graph_title Inode usage (in percent)\n";
     print "graph_args --upper-limit 100 -l 0\n";
     print "graph_vlabel %\n";
     print "graph_scale no\n";
