Index: munin-1.2.6/Makefile
===================================================================
--- munin-1.2.6.orig/Makefile	2008-06-12 10:02:57.000000000 +0200
+++ munin-1.2.6/Makefile	2008-06-12 10:05:06.000000000 +0200
@@ -102,7 +102,7 @@
 	#configure plugins.
 
 install-man: build-man
-	mkdir -p $(MANDIR)/man1 $(MANDIR)/man5 $(MANDIR)/man8
+	mkdir -p $(MANDIR)/man1 $(MANDIR)/man5 $(MANDIR)/man8 $(MANDIR)/man1
 	$(INSTALL) -m 0644 build/doc/munin-node.conf.5 $(MANDIR)/man5/
 	$(INSTALL) -m 0644 build/doc/munin.conf.5 $(MANDIR)/man5/
 	$(INSTALL) -m 0644 build/doc/munin-node.8 $(MANDIR)/man8/
@@ -114,6 +114,7 @@
 	$(INSTALL) -m 0644 build/doc/munin-limits.8 $(MANDIR)/man8/
 	$(INSTALL) -m 0644 build/doc/munin-html.8 $(MANDIR)/man8/
 	$(INSTALL) -m 0644 build/doc/munin-cron.8 $(MANDIR)/man8/
+	$(INSTALL) -m 0644 build/doc/munindoc.1 $(MANDIR)/man1/
 
 install-doc: build-doc
 	mkdir -p $(DOCDIR)
@@ -205,6 +206,8 @@
 		server/munin.conf.pod > build/doc/munin.conf.5
 	pod2man  --section=5 --release=$(RELEASE) --center="Munin Documentation" \
 		node/munin-node.conf.pod > build/doc/munin-node.conf.5
+	pod2man  --section=1 --release=$(RELEASE) --center="Munin Documentation" \
+		build/node/munindoc > build/doc/munindoc.1
 
 	touch build-man-stamp
 
Index: munin-1.2.6/node/munindoc.in
===================================================================
--- munin-1.2.6.orig/node/munindoc.in	2008-06-12 10:06:26.000000000 +0200
+++ munin-1.2.6/node/munindoc.in	2008-06-12 10:07:39.000000000 +0200
@@ -80,3 +80,26 @@
 
 This program displays munin documentation found in pod format adjacent
 to plugins (and later other parts of Munin).
+Please see "man perlpod" for the reference manual to writing pod files.
+
+=head1 AUTHORS
+
+Nicolai Langfeldt
+
+=head1 BUGS
+
+None known.
+
+=head1 COPYRIGHT
+
+Copyright (C) 2008 Nicolai Langfeldt/Linpro AS
+
+This is free software; see the source for copying conditions. There is
+NO warranty; not even for MERCHANTABILITY or FITNESS FOR A PARTICULAR
+PURPOSE.
+
+This program is released under the GNU General Public License
+
+=cut
+
+
