Index: munin-1.2.6/node/node.d/exim_mailstats.in
===================================================================
--- munin-1.2.6.orig/node/node.d/exim_mailstats.in	2008-06-22 22:02:49.000000000 +0200
+++ munin-1.2.6/node/node.d/exim_mailstats.in	2008-06-22 22:03:24.000000000 +0200
@@ -220,7 +220,7 @@
             $rejected++;
         }
     }
-    close($LOGFILE) or die "Could not close $fname after reading: $!";
+    return tail_close($LOGFILE);
 }
 
 # vim:syntax=perl
