Index: munin-1.2.6/Makefile
===================================================================
--- munin-1.2.6.orig/Makefile	2008-06-15 21:41:08.000000000 +0200
+++ munin-1.2.6/Makefile	2008-06-15 21:41:38.000000000 +0200
@@ -66,7 +66,7 @@
 	mkdir -p $(CONFDIR)/plugin-conf.d
 	mkdir -p $(LIBDIR)/plugins
 	mkdir -p $(SBINDIR)
-	mkdir -p $(PERLLIB)/Munin/Plugin
+	mkdir -p $(PERLLIB)/Munin
 
 	mkdir -p $(LOGDIR)
 	mkdir -p $(STATEDIR)
